README.md

```diff
@@ -456,6 +456,7 @@ This list is the result of Pull Requests, reviews, ideas and work done by 1100+
   * [Forestry.io](https://forestry.io/) — Headless CMS. Give your editors the power of Git. Create and edit Markdown-based content with ease. Comes with three free sites that includes 3 editors, Instant Previews. Integrates with blogs hosted on Netlify/GitHubpages/ elsewhere
   * [FrontAid](https://frontaid.io/) — Headless CMS that stores JSON content directly in your own Git repository. No restrictions.
   * [kontent.ai](https://www.kontent.ai) - A Content-as-a-Service platform that gives you all the headless CMS benefits while empowering marketers at the same time. Developer plan provides 2 users with unlimited projects with 2 environments for each, 500 content items, 2 languages with Delivery and Management API, and Custom elements support. Larger plans available to meet your needs.
+  * [PocketHost](https://pockethost.io/) — Hosting for Pocketbase a headless CMS powered by Svelte, Vite, Typescript, and SQLite. Free tier for developers.
```

Comment:
I don’t see any pricing information or anything detailing what’s free and what isn’t.

Reply:
For the moment is full open for testing, but future pricing is under discussion. It will have a free developer tier as you can see in the maintainer's post:
https://github.com/benallfree/pockethost/issues/44

Reply:
Cool. Once that’s visible on the website we can revisit this one. 

You can’t expect us to read long issues for each of thousands of sites to know what the story is. Has to be clear on the site. 

Thanks.